pandas/core/frame.py

```diff
@@ -553,28 +553,28 @@ def __init__(
         index: Optional[Axes] = None,
         columns: Optional[Axes] = None,
         dtype: Optional[Dtype] = None,
-        copy: bool = False,
+        copy: Optional[bool] = None,
     ):
+        orig_copy = copy  # GH#38939
+        copy = copy if copy is not None else False
+
         if data is None:
             data = {}
+            copy = True
```

Comment:
Do you remember what's the reason this is added? (I would assume that if there is no data, copy=True or False shouldn't matter)

Reply:
i dont recall off the top of my head.  will revert and re-run tests to find out, and either remove or add a comment